lib/player.rb

```diff
@@ -0,0 +1,29 @@
+require './lib/ship'
+require './lib/board'
+
+class Player
+  attr_reader :cruiser, :submarine, :board
+
+  def initialize
+    @cruiser = Ship.new("Cruiser", 3)
+    @submarine = Ship.new("Submarine", 2)
+    @board = Board.new
+  end
```

Comment:
Making these instance variables is definitely a risky move, but I think it pays off.